src/main/java/org/jenkinsci/test/acceptance/po/TopLevelItem.java

```diff
@@ -58,8 +58,7 @@ public TopLevelItem(PageObject context, URL url, String name) {
     }
 
     /**
-     * Renames the job. Opens the configuration section, sets the name and saves the form. Finally the rename is
-     * confirmed.
+     * Changes the description. Opens the configuration section, sets the description and saves the form.
```

Comment:
I should have noticed this before but wouldn't it be also better if the method is called setDescription? Or changeDescription? Typically, a method name contains a verb...